Mailer.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace katanyoo\mailgunmailer;
+namespace farmani\mailgunmailer;
```

Comment:
Why would you change the namespace here? This is the `katanyoo` version, not your fork.

Reply:
I even cannot remember what is this. anyway drop this PR I was rewriting another library